web/src/components/ResourcesDialog.tsx

```diff
@@ -27,7 +27,9 @@ const ResourcesDialog: React.FC<Props> = (props: Props) => {
     resources: [],
     isUploadingResource: false,
   });
-
+  const [filename, setFilename] = useState<string>("");
```

Comment:
Maybe we should create a new component to edit resource name. e.g. `ChangeMemoCreatedTsDialog.tsx`

Reply:
Sorry I don't fully understand what you mean. Do you think it would be better to change the filename in a dialog compared to in-place? Or is it just that I need to extract a component that modifies the filename?

Reply:
> change the filename in a dialog

The `ResourcesDialog` is mainly using to display resource list instead of edit it. So I'd like to edit a resource information including filename in aother dialog.

Reply:
Sounds good, I'll revise and commit later.

Reply:
https://user-images.githubusercontent.com/31177490/198816163-195e004d-1eda-4175-8437-dd101b88b849.mp4

Reply:
Why my latest commits cannot be attached to this PR automatically ? Orz